.github/workflows/configure.sh

```diff
@@ -1,37 +1,42 @@
-#!/bin/bash -e 
+#!/bin/bash -xe
 
 maker=$1
+device=$2
 
 if [ "${maker}" = "cmake" ]; then
     rm -rf build
     mkdir -p build
 fi
 
-mydir=`dirname $0`
-source $mydir/setup_env.sh
+mydir=$(dirname $0)
+source ${mydir}/setup_env.sh
 
-section "======================================== setup build"
+section "======================================== Verify dependencies"
+quiet module list
+quiet which g++
+quiet g++ --version
 
+echo "MKLROOT=${MKLROOT}"
+
+section "======================================== Environment"
+env
+
+section "======================================== Setup build"
 export color=no
 rm -rf ${top}/install
 if [ "${maker}" = "make" ]; then
     make distclean
     make config CXXFLAGS="-Werror" prefix=${top}/install
 fi
```

Comment:
I am unfamiliar with lapackpp build process but is it normal that the `cmake` case clone `blaspp` and not the `make` case?

Reply:
Yes, the makefile will automatically build blaspp in the blaspp subdirectory if the location of blaspp isn't provided.

Reply:
Yes, the LAPACK++ `configure.py` automatically downloads and builds BLAS++, while with CMake we require that BLAS++ is already installed. It's feasible for CMake to also download BLAS++, as it does download TestSweeper if needed, but we haven't done that.